L1Trigger/L1THGCal/interface/HGCalCoarseTriggerCellMapping.h

```diff
@@ -58,6 +58,10 @@ class HGCalCoarseTriggerCellMapping {
   static constexpr int kHGCalCellMaskV9Inv_ = ~0xff;
   static constexpr int kHGCalScinCellMaskInv_ = ~0x1ffff;
 
+  static constexpr int kRocNum1_ = 1;
```

Comment:
Since writing 1=1, 2=2, 3=3 doesn't bring any additional information, it might be better to distinguish the HGCROCs by their relative rotation angles, e.g.: `kRoc0deg`, `kRoc120deg`, `kRoc240deg`